packages/skia/cpp/api/JsiSkImage.h

```diff
@@ -186,6 +186,14 @@ class JsiSkImage : public JsiSkWrappingSkPtrHostObject<SkImage> {
         runtime, std::make_shared<JsiSkImage>(getContext(), std::move(image)));
   }
 
+  JSI_HOST_FUNCTION(isTextureBacked) {
+    return static_cast<bool>(getObject()->isTextureBacked());
+  }
+
+  JSI_HOST_FUNCTION(textureSize) {
+    return static_cast<double>(getObject()->textureSize());
```

Comment:
What does this do?

Reply:
Usefull to have an idea how [much memory your textures use](https://api.skia.org/classSkImage.html#acd7bda080bbe286f56fb0fe7b8af30c7). In the future, we could allow users to configure skia memory [cache size](https://api.skia.org/classGrDirectContext.html#af972a5f50215ee314c48e5749d6259b9) so you can then calculate also for each texture the pressure and also call purge when you need.

For now added it so we can collect info on how much GPU memory we used by our textures.